Blazor Preview 9 updates (#14157)
aspnet/AspNetCore.Docs: aspnetcore/blazor/routing.md

> The `Router` component enables routing, and a route template is provided to each accessible component. The `Router` component appears in the *App.razor* file:

Reviewer: ```suggestion
The `Router` component enables routing to each component with a specified route. The `Router` component appears in the *App.razor* file:
```

</Router>
```

When a *.razor* file with an `@page` directive is compiled, the generated class is provided a <xref:Microsoft.AspNetCore.Mvc.RouteAttribute> specifying the route template.

At runtime, the `RouteView` component:

* Receives the `RouteData` from the `Router` along with any desired parameters.

> * Searches for component classes with a `RouteAttribute` and renders the component with a route template that matches the requested URL.

Reviewer: The `RouteView` component doesn't search for components. The `RouteData` passed to the `RouteView` from the `Router` already specifies a specific component and the matched parameters. All the `RouteView` does is render the specified component with its layout (or an optional default layout) using the specified parameters.
